doc/tech/README.md

```diff
@@ -0,0 +1,148 @@
+[<< Back](https://cntt-n.github.io/CNTT/)
+# Common NFVI Telco Taskforce
+
+## Table of Contents
+* [1. Overview](#1.0)
+* [2. Scope](#2.0)
+* [3. Principles](#3.0)
+* [4. Technology Direction](#4.0)
+  * [4.1 IO Virtualisation](#4.1)
+  * [4.2 SmartNICs](#4.2)
+  * [4.3 Acceleration Cards](#4.3)
+  * [4.4 FPGA](#4.4)
+  * [4.5 GPUs/NPUs](#4.5)
+  * [4.6 EPA/NFD](#4.6)
+* [5. Available Specifications](#5.0)
+
+<a name="1.0"></a>
+## 1. Overview
+
+Initially organized early in 2019, the Common Network Function Virtualisation Infrastructure Telecom Taskforce (CNTT) was initially created in response to rapid changes in how networking applications are being designed, built and managed, plus a growing recognition of a perceived functional gap between the previous standard infrastructure models and the architectures needed to support Network Function Virtualization (NFV) applications.  Organizationally the Common Telco Network Function Virtualisation Infrastructure (NFVI)  project, jointly hosted by GSMA and the Linux Foundation, operates as an open committee responsible for creating and documenting an industry aligned Common NFVI Framework.  The CNTT group was created with the intent that it would create the NFVI framework, and eventually morph into an on-going project under the auspices of the GSMA and the Linux Foundation umbrellas.  The final on-going operational form of the group will be determined as the project evolves. 
+
+<a name="2.0"></a>
+## 2. Scope
+
+Within the framework of the Common Telecom NFVI vision, there are three levels of documents needed to document the components and allow the practical application of the systems. They are, as highlighted in **Figure 1**:  **Reference Model**, **Reference Architecture**, **Reference Implementation**, and **Reference Certification**.
+
+<p align="center"><img src="./figures/tech_scope.png" alt="scope" title="Document Types" width="100%"/></p>
+<p align="center"><b>Figure 1:</b> Scope of CNTT</p>
+
+<a name="2.1"></a>
+## 2.1 Specification Types
+
+- **Reference Model (RM)**: focuses on the __**Infrastructure Abstraction**__ and how services and resources are exposed to VNFs/CNFs. It written at a high enough level that as new **Reference Architectures** and **Reference Implementations** are added, the model document should require few or no changes. Additionally, the Reference Model is intended to be neutral towards VMs or Containers.
```

Comment:
Possible suggested reword:
**Reference Model (RM)**: focuses on the __**NFVI Abstraction**__ and how services and resources are exposed to VNFs/CNFs, and external systems such as VNF/CNF orchestrators and NFV managers. It is an abstract and technology agnostic document which should require few or no changes as new technologies that require new **Reference Architectures** are introduced . For example, the Reference Model is neutral on the VNF/CNF execution environments, i.e., Virtual Machines or Containers.

Reply:
@pgoyal01 if you are OK with it, I will keep the definitions as is since this PR is not proposing to change this part (it was a simple move from RM to here).